screens/SendSetupScreen.js

```diff
@@ -295,7 +324,143 @@ const SendSetupScreen = ({
     }
   };
 
+  const handleDeepLink = ({
+    address,
+    amount,
+    tokenAmount,
+    label,
+    tokenId,
+    symbol
+  }): DeepLinkParams => {
```

Comment:
DeepLinkParams could use an update.
`tokenAmount`, `tokenId`, and `symbol` are missing from it

Reply:
those fields are needed for `prefillSLP()`.

the deeplinkparams works for both bch and slp.

Reply:
They should be added as optional parameters in the flow type in that case?